package.json

```diff
@@ -14,7 +14,7 @@
     "test": "TZ=UTC react-scripts test --silent",
     "eject": "react-scripts eject",
     "test:generators": "ts-node --project=./internals/ts-node.tsconfig.json ./internals/testing/generators/test-generators.ts",
-    "start:prod": "yarn run build && serve -s build",
+    "start:prod": "yarn run build && serve -c ./internals/serve-with-csp.json --single --no-etag",
```

Comment:
`-s` expanded to `--single` here. from the docs

>       -s, --single                        Rewrite all not-found requests to `index.html`

the `build` dir is moved to that .json file

`--no-etag` is added :shrug: shouldn't hurt. although would it be okay to move this into the config json file? although that would start making the scope of that config file creep. maybe we should put everything in there an call it something like 'serve-prod.json'

Reply:
Yeah.. Ideally I'd still like CSP to be in it's own file for searchability

Reply:
oh

Reply:
Will refactor in a separate PR